### ISSUE TITLE
How to add new language

### ISSUE THREAD

**Issue by qborreda** [tinganho/l10ns#155]
Hi @tinganho 

On an app I already have working with the use of `l10ns`, we want to implement a new language. 
I have included an entry on the configuration file for the new language (pt) like
```json
"languages": {
  "en-US": "English (US)",
  "es-ES": "Spanish (Spain)",
  "pt-BR": "Portuguese (Brazil)"
},
```
and we sent one of the used json files to be translated, so now we have a new `pt-BR.json` with the same entries and translations to Portuguese .. 

Then again, `l10ns interface` refuses to load, and `l10ns compile` seems to do nothing at all.
I even tried to rename `pt-BR.json`away, and let l10ns generate it's own empty json, but it didn't work.

I've just updated to `l10ns@2.4.9` installed globally .. and using node v7.7.x here .. 

Is there anything I'm doing wrong or missing?

**Comment by qborreda** [comment]
Ok, node throws a deprecation message when trying to launch `l10ns interface`
`(node:21530) DeprecationWarning: 'GLOBAL' is deprecated, use 'global'`

I've tried to remove my changes, and indeed, it's not working as it used to .. maybe that's the cause

**Comment by qborreda** [comment]
I've tried with both `node 6.9` and `node 6.3`, and I'm getting the same, interface not opening the webpage ..

**Comment by tinganho** [comment]
Is it showing any errors or just stalling? Could you try ti visit the link as well to make sure it is nothing with opening the browser.

**Comment by qborreda** [comment]
`l10ns interface` launches the server, opens up a new tab, stalls showing white page .. no errors shown on the console.

**Comment by tinganho** [comment]
I will try to take a look at this problem later today.

**Comment by qborreda** [comment]
Is there anything you'd recommend me checking meanwhile?

**Comment by tinganho** [comment]
hmm, I tried to reproduce the error now. And it seems to work for me. I just added `pt-BR` to my l10ns.json file. 

Could you try to remove pt-BR  and its corresponding `pt-BR.json` file and see if it works?

**Comment by qborreda** [comment]
I did, yes .. to no avail .. in fact, it stopped working as it were before trying to add the new language, which is worrying .. 

Here is the `l10ns.json` before
```json
{
  "defaultProject": "backoffice",
  "projects": {
    "backoffice": {
      "timezones": [
        "America/Los_Angeles",
        "Europe/Madrid"
      ],
      "currencies": [
        "USD",
        "EUR"
      ],
      "interface": {
        "autoOpen": true,
        "port": 3030
      },
      "languages": {
        "en-US": "English (US)",
        "es-ES": "Spanish (Spain)"
      },
      "defaultLanguage": "en-US",
      "programmingLanguage": "javascript",
      "store": "src/localizations/",
      "output": "src/localizations/output",
      "source": [
        "src/localizations/dictionary.js"
      ]
    }
  }
}
```
We are using just one `dictionary.js` file with all the string identifiers, because we're using ES6 in our react code .. 

I can't even make this one work anymore /sadface

**Comment by tinganho** [comment]
OK I could reproduce it now after I reinstalled l10ns. Then it must be problem downstream perhaps. I will take a closer look later this day.

**Comment by qborreda** [comment]
More data. I've conducted some experiments at a peep computer running `2.4.8`, and everything worked nicely with both `node 7.7.3` and even `node 4.4.0` ..

**Comment by tinganho** [comment]
Ok I found the problem now. I will push a fix soon.

**Comment by qborreda** [comment]
Bravo! Thank you mate!

**Comment by tinganho** [comment]
Now, it should be fixed in 2.4.10.

**Comment by qborreda** [comment]
Working with `2.4.10`  ... thank you man!
